xml/http_wwwauth.xml

```diff
@@ -106,6 +172,13 @@
     <param pos="1" name="os.product"/>
     <param pos="2" name="host.mac"/>
   </fingerprint>
+  <fingerprint pattern="^(?:Basic|Digest).*realm=&quot;Thomson(?: Gateway)?&quot;.*$">
+    <description>Thomson generic devices</description>
+    <example>Digest realm=&quot;Thomson Gateway&quot;</example>
+    <example>Basic realm=&quot;Thomson&quot;</example>
+    <param pos="0" name="hw.vendor" value="Thomson"/>
+    <param pos="0" name="hw.device" value="Broadband router"/>
```

Comment:
Another instance of inconsistent capitalization in the project.  My preference is `Broadband Router` but consistency is more important than my preference. ;)

```
http_wwwauth.xml:    <param pos="0" name="os.device" value="Broadband router"/>
http_wwwauth.xml:    <param pos="0" name="os.device" value="Broadband router"/>
http_wwwauth.xml:    <param pos="0" name="os.device" value="Broadband router"/>
http_wwwauth.xml:    <param pos="0" name="os.device" value="Broadband router"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Broadband Router"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Broadband Router"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Broadband router"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Broadband Router"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Broadband Router"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Broadband Router"/>
telnet_banners.xml:    <param pos="0" name="hw.device" value="Broadband Router"/>
telnet_banners.xml:    <param pos="0" name="hw.device" value="Broadband Router"/>
telnet_banners.xml:    <param pos="0" name="os.device" value="Broadband router"/>
telnet_banners.xml:    <param pos="0" name="os.device" value="Broadband router"/>
```